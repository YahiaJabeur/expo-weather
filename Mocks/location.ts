export const location = {
  id: 568120,
  name: "Berlin",
  region: "Berlin",
  country: "Germany",
  lat: 52.52,
  lon: 13.4,
  url: "berlin-berlin-germany",
};
