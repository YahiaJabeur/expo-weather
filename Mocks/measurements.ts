export const measurements = [
  {
    time_epoch: 1717448400,
    time: "2024-06-04 00:00",
    temp_c: 15.5,
    temp_f: 60,
    is_day: 0,
    condition: {
      text: "Clear ",
      icon: "//cdn.weatherapi.com/weather/64x64/night/113.png",
      code: 1000,
    },
    wind_mph: 3.4,
    wind_kph: 5.4,
    wind_degree: 244,
    wind_dir: "WSW",
    pressure_mb: 1016,
    pressure_in: 29.99,
    precip_mm: 0,
    precip_in: 0,
    snow_cm: 0,
    humidity: 56,
    cloud: 12,
    feelslike_c: 15.6,
    feelslike_f: 60,
    windchill_c: 15.6,
    windchill_f: 60,
    heatindex_c: 15.6,
    heatindex_f: 60,
    dewpoint_c: 6.8,
    dewpoint_f: 44.2,
    will_it_rain: 0,
    chance_of_rain: 0,
    will_it_snow: 0,
    chance_of_snow: 0,
    vis_km: 10,
    vis_miles: 6,
    gust_mph: 7,
    gust_kph: 11.3,
    uv: 1,
  },
  {
    time_epoch: 1717452000,
    time: "2024-06-04 01:00",
    temp_c: 14.5,
    temp_f: 58,
    is_day: 0,
    condition: {
      text: "Clear ",
      icon: "//cdn.weatherapi.com/weather/64x64/night/113.png",
      code: 1000,
    },
    wind_mph: 3.6,
    wind_kph: 5.8,
    wind_degree: 242,
    wind_dir: "WSW",
    pressure_mb: 1015,
    pressure_in: 29.98,
    precip_mm: 0,
    precip_in: 0,
    snow_cm: 0,
    humidity: 53,
    cloud: 15,
    feelslike_c: 14.7,
    feelslike_f: 58.4,
    windchill_c: 14.7,
    windchill_f: 58.4,
    heatindex_c: 14.5,
    heatindex_f: 58,
    dewpoint_c: 5,
    dewpoint_f: 41.1,
    will_it_rain: 0,
    chance_of_rain: 0,
    will_it_snow: 0,
    chance_of_snow: 0,
    vis_km: 10,
    vis_miles: 6,
    gust_mph: 7.5,
    gust_kph: 12.1,
    uv: 1,
  },
  {
    time_epoch: 1717455600,
    time: "2024-06-04 02:00",
    temp_c: 14.2,
    temp_f: 57.6,
    is_day: 0,
    condition: {
      text: "Clear ",
      icon: "//cdn.weatherapi.com/weather/64x64/night/113.png",
      code: 1000,
    },
    wind_mph: 2.5,
    wind_kph: 4,
    wind_degree: 206,
    wind_dir: "SSW",
    pressure_mb: 1015,
    pressure_in: 29.96,
    precip_mm: 0,
    precip_in: 0,
    snow_cm: 0,
    humidity: 50,
    cloud: 3,
    feelslike_c: 14.8,
    feelslike_f: 58.6,
    windchill_c: 14.8,
    windchill_f: 58.6,
    heatindex_c: 14.2,
    heatindex_f: 57.6,
    dewpoint_c: 4.1,
    dewpoint_f: 39.3,
    will_it_rain: 0,
    chance_of_rain: 0,
    will_it_snow: 0,
    chance_of_snow: 0,
    vis_km: 10,
    vis_miles: 6,
    gust_mph: 5.2,
    gust_kph: 8.3,
    uv: 1,
  },
  {
    time_epoch: 1717459200,
    time: "2024-06-04 03:00",
    temp_c: 14.2,
    temp_f: 57.6,
    is_day: 0,
    condition: {
      text: "Clear ",
      icon: "//cdn.weatherapi.com/weather/64x64/night/113.png",
      code: 1000,
    },
    wind_mph: 3.6,
    wind_kph: 5.8,
    wind_degree: 238,
    wind_dir: "WSW",
    pressure_mb: 1015,
    pressure_in: 29.96,
    precip_mm: 0,
    precip_in: 0,
    snow_cm: 0,
    humidity: 48,
    cloud: 3,
    feelslike_c: 14.4,
    feelslike_f: 57.8,
    windchill_c: 14.4,
    windchill_f: 57.8,
    heatindex_c: 14.2,
    heatindex_f: 57.6,
    dewpoint_c: 3.5,
    dewpoint_f: 38.2,
    will_it_rain: 0,
    chance_of_rain: 0,
    will_it_snow: 0,
    chance_of_snow: 0,
    vis_km: 10,
    vis_miles: 6,
    gust_mph: 7.5,
    gust_kph: 12.1,
    uv: 1,
  },
  {
    time_epoch: 1717462800,
    time: "2024-06-04 04:00",
    temp_c: 14,
    temp_f: 57.2,
    is_day: 0,
    condition: {
      text: "Clear ",
      icon: "//cdn.weatherapi.com/weather/64x64/night/113.png",
      code: 1000,
    },
    wind_mph: 3.6,
    wind_kph: 5.8,
    wind_degree: 237,
    wind_dir: "WSW",
    pressure_mb: 1014,
    pressure_in: 29.95,
    precip_mm: 0,
    precip_in: 0,
    snow_cm: 0,
    humidity: 48,
    cloud: 2,
    feelslike_c: 14.1,
    feelslike_f: 57.4,
    windchill_c: 14.1,
    windchill_f: 57.4,
    heatindex_c: 14,
    heatindex_f: 57.2,
    dewpoint_c: 3.2,
    dewpoint_f: 37.7,
    will_it_rain: 0,
    chance_of_rain: 0,
    will_it_snow: 0,
    chance_of_snow: 0,
    vis_km: 10,
    vis_miles: 6,
    gust_mph: 7.5,
    gust_kph: 12.1,
    uv: 1,
  },
  {
    time_epoch: 1717466400,
    time: "2024-06-04 05:00",
    temp_c: 13.6,
    temp_f: 56.5,
    is_day: 0,
    condition: {
      text: "Clear ",
      icon: "//cdn.weatherapi.com/weather/64x64/night/113.png",
      code: 1000,
    },
    wind_mph: 3.4,
    wind_kph: 5.4,
    wind_degree: 240,
    wind_dir: "WSW",
    pressure_mb: 1015,
    pressure_in: 29.96,
    precip_mm: 0,
    precip_in: 0,
    snow_cm: 0,
    humidity: 49,
    cloud: 0,
    feelslike_c: 13.8,
    feelslike_f: 56.8,
    windchill_c: 13.8,
    windchill_f: 56.8,
    heatindex_c: 13.6,
    heatindex_f: 56.6,
    dewpoint_c: 3.1,
    dewpoint_f: 37.7,
    will_it_rain: 0,
    chance_of_rain: 0,
    will_it_snow: 0,
    chance_of_snow: 0,
    vis_km: 10,
    vis_miles: 6,
    gust_mph: 7,
    gust_kph: 11.3,
    uv: 1,
  },
  {
    time_epoch: 1717470000,
    time: "2024-06-04 06:00",
    temp_c: 14.1,
    temp_f: 57.4,
    is_day: 0,
    condition: {
      text: "Clear ",
      icon: "//cdn.weatherapi.com/weather/64x64/night/113.png",
      code: 1000,
    },
    wind_mph: 3.8,
    wind_kph: 6.1,
    wind_degree: 241,
    wind_dir: "WSW",
    pressure_mb: 1015,
    pressure_in: 29.98,
    precip_mm: 0,
    precip_in: 0,
    snow_cm: 0,
    humidity: 54,
    cloud: 0,
    feelslike_c: 14.2,
    feelslike_f: 57.5,
    windchill_c: 14.2,
    windchill_f: 57.5,
    heatindex_c: 14.1,
    heatindex_f: 57.4,
    dewpoint_c: 5,
    dewpoint_f: 40.9,
    will_it_rain: 0,
    chance_of_rain: 0,
    will_it_snow: 0,
    chance_of_snow: 0,
    vis_km: 10,
    vis_miles: 6,
    gust_mph: 8,
    gust_kph: 12.9,
    uv: 1,
  },
  {
    time_epoch: 1717473600,
    time: "2024-06-04 07:00",
    temp_c: 17.5,
    temp_f: 63.4,
    is_day: 1,
    condition: {
      text: "Sunny",
      icon: "//cdn.weatherapi.com/weather/64x64/day/113.png",
      code: 1000,
    },
    wind_mph: 3.8,
    wind_kph: 6.1,
    wind_degree: 245,
    wind_dir: "WSW",
    pressure_mb: 1015,
    pressure_in: 29.97,
    precip_mm: 0,
    precip_in: 0,
    snow_cm: 0,
    humidity: 50,
    cloud: 8,
    feelslike_c: 17.5,
    feelslike_f: 63.4,
    windchill_c: 17.5,
    windchill_f: 63.4,
    heatindex_c: 17.5,
    heatindex_f: 63.4,
    dewpoint_c: 7,
    dewpoint_f: 44.5,
    will_it_rain: 0,
    chance_of_rain: 0,
    will_it_snow: 0,
    chance_of_snow: 0,
    vis_km: 10,
    vis_miles: 6,
    gust_mph: 7.4,
    gust_kph: 11.8,
    uv: 5,
  },
  {
    time_epoch: 1717477200,
    time: "2024-06-04 08:00",
    temp_c: 20.5,
    temp_f: 68.9,
    is_day: 1,
    condition: {
      text: "Sunny",
      icon: "//cdn.weatherapi.com/weather/64x64/day/113.png",
      code: 1000,
    },
    wind_mph: 4.7,
    wind_kph: 7.6,
    wind_degree: 242,
    wind_dir: "WSW",
    pressure_mb: 1015,
    pressure_in: 29.97,
    precip_mm: 0,
    precip_in: 0,
    snow_cm: 0,
    humidity: 42,
    cloud: 2,
    feelslike_c: 20.5,
    feelslike_f: 68.9,
    windchill_c: 20.5,
    windchill_f: 68.9,
    heatindex_c: 20.5,
    heatindex_f: 68.9,
    dewpoint_c: 7.3,
    dewpoint_f: 45.1,
    will_it_rain: 0,
    chance_of_rain: 0,
    will_it_snow: 0,
    chance_of_snow: 0,
    vis_km: 10,
    vis_miles: 6,
    gust_mph: 6.3,
    gust_kph: 10.1,
    uv: 6,
  },
  {
    time_epoch: 1717480800,
    time: "2024-06-04 09:00",
    temp_c: 22.7,
    temp_f: 72.8,
    is_day: 1,
    condition: {
      text: "Sunny",
      icon: "//cdn.weatherapi.com/weather/64x64/day/113.png",
      code: 1000,
    },
    wind_mph: 5.6,
    wind_kph: 9,
    wind_degree: 238,
    wind_dir: "WSW",
    pressure_mb: 1015,
    pressure_in: 29.97,
    precip_mm: 0,
    precip_in: 0,
    snow_cm: 0,
    humidity: 38,
    cloud: 0,
    feelslike_c: 24.1,
    feelslike_f: 75.4,
    windchill_c: 22.7,
    windchill_f: 72.8,
    heatindex_c: 24.1,
    heatindex_f: 75.4,
    dewpoint_c: 7.6,
    dewpoint_f: 45.6,
    will_it_rain: 0,
    chance_of_rain: 0,
    will_it_snow: 0,
    chance_of_snow: 0,
    vis_km: 10,
    vis_miles: 6,
    gust_mph: 6.5,
    gust_kph: 10.4,
    uv: 6,
  },
  {
    time_epoch: 1717484400,
    time: "2024-06-04 10:00",
    temp_c: 24.6,
    temp_f: 76.2,
    is_day: 1,
    condition: {
      text: "Sunny",
      icon: "//cdn.weatherapi.com/weather/64x64/day/113.png",
      code: 1000,
    },
    wind_mph: 6.9,
    wind_kph: 11.2,
    wind_degree: 246,
    wind_dir: "WSW",
    pressure_mb: 1015,
    pressure_in: 29.97,
    precip_mm: 0,
    precip_in: 0,
    snow_cm: 0,
    humidity: 33,
    cloud: 3,
    feelslike_c: 24.7,
    feelslike_f: 76.4,
    windchill_c: 24.6,
    windchill_f: 76.2,
    heatindex_c: 24.7,
    heatindex_f: 76.4,
    dewpoint_c: 7.2,
    dewpoint_f: 44.9,
    will_it_rain: 0,
    chance_of_rain: 0,
    will_it_snow: 0,
    chance_of_snow: 0,
    vis_km: 10,
    vis_miles: 6,
    gust_mph: 8,
    gust_kph: 12.9,
    uv: 6,
  },
  {
    time_epoch: 1717488000,
    time: "2024-06-04 11:00",
    temp_c: 25.8,
    temp_f: 78.5,
    is_day: 1,
    condition: {
      text: "Sunny",
      icon: "//cdn.weatherapi.com/weather/64x64/day/113.png",
      code: 1000,
    },
    wind_mph: 7.6,
    wind_kph: 12.2,
    wind_degree: 253,
    wind_dir: "WSW",
    pressure_mb: 1015,
    pressure_in: 29.96,
    precip_mm: 0,
    precip_in: 0,
    snow_cm: 0,
    humidity: 30,
    cloud: 6,
    feelslike_c: 25.2,
    feelslike_f: 77.4,
    windchill_c: 25.8,
    windchill_f: 78.5,
    heatindex_c: 25.2,
    heatindex_f: 77.4,
    dewpoint_c: 6.7,
    dewpoint_f: 44.1,
    will_it_rain: 0,
    chance_of_rain: 0,
    will_it_snow: 0,
    chance_of_snow: 0,
    vis_km: 10,
    vis_miles: 6,
    gust_mph: 8.7,
    gust_kph: 14.1,
    uv: 7,
  },
  {
    time_epoch: 1717491600,
    time: "2024-06-04 12:00",
    temp_c: 26.7,
    temp_f: 80,
    is_day: 1,
    condition: {
      text: "Sunny",
      icon: "//cdn.weatherapi.com/weather/64x64/day/113.png",
      code: 1000,
    },
    wind_mph: 7.6,
    wind_kph: 12.2,
    wind_degree: 252,
    wind_dir: "WSW",
    pressure_mb: 1014,
    pressure_in: 29.96,
    precip_mm: 0,
    precip_in: 0,
    snow_cm: 0,
    humidity: 28,
    cloud: 5,
    feelslike_c: 25.7,
    feelslike_f: 78.2,
    windchill_c: 26.7,
    windchill_f: 80,
    heatindex_c: 25.7,
    heatindex_f: 78.2,
    dewpoint_c: 6.6,
    dewpoint_f: 43.9,
    will_it_rain: 0,
    chance_of_rain: 0,
    will_it_snow: 0,
    chance_of_snow: 0,
    vis_km: 10,
    vis_miles: 6,
    gust_mph: 8.7,
    gust_kph: 14.1,
    uv: 7,
  },
  {
    time_epoch: 1717495200,
    time: "2024-06-04 13:00",
    temp_c: 28.5,
    temp_f: 83.3,
    is_day: 1,
    condition: {
      text: "Sunny",
      icon: "//cdn.weatherapi.com/weather/64x64/day/113.png",
      code: 1000,
    },
    wind_mph: 7.8,
    wind_kph: 12.6,
    wind_degree: 243,
    wind_dir: "WSW",
    pressure_mb: 1014,
    pressure_in: 29.94,
    precip_mm: 0,
    precip_in: 0,
    snow_cm: 0,
    humidity: 26,
    cloud: 5,
    feelslike_c: 27,
    feelslike_f: 80.7,
    windchill_c: 28.5,
    windchill_f: 83.3,
    heatindex_c: 27,
    heatindex_f: 80.7,
    dewpoint_c: 7.1,
    dewpoint_f: 44.8,
    will_it_rain: 0,
    chance_of_rain: 0,
    will_it_snow: 0,
    chance_of_snow: 0,
    vis_km: 10,
    vis_miles: 6,
    gust_mph: 9,
    gust_kph: 14.5,
    uv: 7,
  },
  {
    time_epoch: 1717498800,
    time: "2024-06-04 14:00",
    temp_c: 29,
    temp_f: 84.2,
    is_day: 1,
    condition: {
      text: "Sunny",
      icon: "//cdn.weatherapi.com/weather/64x64/day/113.png",
      code: 1000,
    },
    wind_mph: 8.7,
    wind_kph: 14,
    wind_degree: 242,
    wind_dir: "WSW",
    pressure_mb: 1014,
    pressure_in: 29.93,
    precip_mm: 0,
    precip_in: 0,
    snow_cm: 0,
    humidity: 25,
    cloud: 4,
    feelslike_c: 27.5,
    feelslike_f: 81.4,
    windchill_c: 29,
    windchill_f: 84.3,
    heatindex_c: 27.5,
    heatindex_f: 81.4,
    dewpoint_c: 7.3,
    dewpoint_f: 45.1,
    will_it_rain: 0,
    chance_of_rain: 0,
    will_it_snow: 0,
    chance_of_snow: 0,
    vis_km: 10,
    vis_miles: 6,
    gust_mph: 10,
    gust_kph: 16.2,
    uv: 7,
  },
  {
    time_epoch: 1717502400,
    time: "2024-06-04 15:00",
    temp_c: 28.8,
    temp_f: 83.9,
    is_day: 1,
    condition: {
      text: "Sunny",
      icon: "//cdn.weatherapi.com/weather/64x64/day/113.png",
      code: 1000,
    },
    wind_mph: 10.3,
    wind_kph: 16.6,
    wind_degree: 244,
    wind_dir: "WSW",
    pressure_mb: 1014,
    pressure_in: 29.93,
    precip_mm: 0,
    precip_in: 0,
    snow_cm: 0,
    humidity: 26,
    cloud: 5,
    feelslike_c: 27.4,
    feelslike_f: 81.2,
    windchill_c: 28.8,
    windchill_f: 83.9,
    heatindex_c: 27.4,
    heatindex_f: 81.2,
    dewpoint_c: 7.7,
    dewpoint_f: 45.8,
    will_it_rain: 0,
    chance_of_rain: 0,
    will_it_snow: 0,
    chance_of_snow: 0,
    vis_km: 10,
    vis_miles: 6,
    gust_mph: 11.8,
    gust_kph: 19,
    uv: 7,
  },
  {
    time_epoch: 1717506000,
    time: "2024-06-04 16:00",
    temp_c: 28.3,
    temp_f: 83,
    is_day: 1,
    condition: {
      text: "Sunny",
      icon: "//cdn.weatherapi.com/weather/64x64/day/113.png",
      code: 1000,
    },
    wind_mph: 10.7,
    wind_kph: 17.3,
    wind_degree: 251,
    wind_dir: "WSW",
    pressure_mb: 1013,
    pressure_in: 29.93,
    precip_mm: 0,
    precip_in: 0,
    snow_cm: 0,
    humidity: 27,
    cloud: 6,
    feelslike_c: 27,
    feelslike_f: 80.6,
    windchill_c: 28.3,
    windchill_f: 83,
    heatindex_c: 27,
    heatindex_f: 80.6,
    dewpoint_c: 7.8,
    dewpoint_f: 46,
    will_it_rain: 0,
    chance_of_rain: 0,
    will_it_snow: 0,
    chance_of_snow: 0,
    vis_km: 10,
    vis_miles: 6,
    gust_mph: 12.3,
    gust_kph: 19.9,
    uv: 7,
  },
  {
    time_epoch: 1717509600,
    time: "2024-06-04 17:00",
    temp_c: 27.3,
    temp_f: 81.1,
    is_day: 1,
    condition: {
      text: "Sunny",
      icon: "//cdn.weatherapi.com/weather/64x64/day/113.png",
      code: 1000,
    },
    wind_mph: 8.1,
    wind_kph: 13,
    wind_degree: 251,
    wind_dir: "WSW",
    pressure_mb: 1013,
    pressure_in: 29.92,
    precip_mm: 0,
    precip_in: 0,
    snow_cm: 0,
    humidity: 29,
    cloud: 7,
    feelslike_c: 26.2,
    feelslike_f: 79.2,
    windchill_c: 27.3,
    windchill_f: 81.1,
    heatindex_c: 26.2,
    heatindex_f: 79.2,
    dewpoint_c: 7.6,
    dewpoint_f: 45.6,
    will_it_rain: 0,
    chance_of_rain: 0,
    will_it_snow: 0,
    chance_of_snow: 0,
    vis_km: 10,
    vis_miles: 6,
    gust_mph: 9.3,
    gust_kph: 14.9,
    uv: 7,
  },
  {
    time_epoch: 1717513200,
    time: "2024-06-04 18:00",
    temp_c: 25.7,
    temp_f: 78.3,
    is_day: 1,
    condition: {
      text: "Sunny",
      icon: "//cdn.weatherapi.com/weather/64x64/day/113.png",
      code: 1000,
    },
    wind_mph: 6,
    wind_kph: 9.7,
    wind_degree: 246,
    wind_dir: "WSW",
    pressure_mb: 1014,
    pressure_in: 29.93,
    precip_mm: 0,
    precip_in: 0,
    snow_cm: 0,
    humidity: 36,
    cloud: 7,
    feelslike_c: 25.6,
    feelslike_f: 78.1,
    windchill_c: 25.7,
    windchill_f: 78.3,
    heatindex_c: 25.6,
    heatindex_f: 78.1,
    dewpoint_c: 9.6,
    dewpoint_f: 49.3,
    will_it_rain: 0,
    chance_of_rain: 0,
    will_it_snow: 0,
    chance_of_snow: 0,
    vis_km: 10,
    vis_miles: 6,
    gust_mph: 7,
    gust_kph: 11.2,
    uv: 7,
  },
];
