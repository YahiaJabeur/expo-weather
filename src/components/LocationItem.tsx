import React from "react";
import { Text, TouchableOpacity, View } from "react-native";
import { createStyleSheet, useStyles } from "react-native-unistyles";

import { City } from "@/types/City";

interface Props {
  item: City;
  onPress: (locationUrl: string) => Promise<void>;
  testID?: string;
}

export const LocationItem = ({
  item: { name, region, country, url },
  onPress,
  testID,
}: Props) => {
  const { styles } = useStyles(stylesheet);

  return (
    <TouchableOpacity
      onPress={() => onPress(url)}
      style={styles.container}
      testID={testID}
      accessibilityLabel={`Location: ${name}, ${region}, ${country}`}
    >
      <View style={styles.nameContainer}>
        <Text style={styles.name}>{name}</Text>
        <Text style={styles.region}>{region}</Text>
      </View>
      <Text style={styles.country}>{country}</Text>
    </TouchableOpacity>
  );
};

const stylesheet = createStyleSheet((theme) => ({
  container: {
    flexDirection: "row",
    borderBottomWidth: 1,
    borderColor: theme.colors.lightGray,
    paddingHorizontal: theme.paddings.lg,
    paddingVertical: theme.paddings.xl,
    justifyContent: "space-between",
  },
  nameContainer: {
    flexDirection: "column",
    marginRight: theme.margins.md,
  },
  name: {
    fontWeight: "bold",
    fontSize: theme.typography.lg,
    color: theme.colors.typography,
    marginBottom: theme.margins.lg,
  },
  region: {
    color: theme.colors.gray,
    alignSelf: "flex-start",
  },
  country: {
    color: theme.colors.primaryLight,
  },
}));
