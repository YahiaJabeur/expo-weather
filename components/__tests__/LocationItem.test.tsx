import React from "react";
import renderer from "react-test-renderer";
import { LocationItem } from "../LocationItem";
import { location } from "@/Mocks/location";

const mockOnPress = jest.fn();

test("renders correctly", () => {
  const tree = renderer
    .create(<LocationItem item={location} onPress={mockOnPress} />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
