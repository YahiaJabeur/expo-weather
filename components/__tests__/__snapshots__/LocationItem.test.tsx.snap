// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessibilityLabel="Location: Berlin, Berlin, Germany"
  accessibilityState={
    {
      "busy": undefined,
      "checked": undefined,
      "disabled": undefined,
      "expanded": undefined,
      "selected": undefined,
    }
  }
  accessibilityValue={
    {
      "max": undefined,
      "min": undefined,
      "now": undefined,
      "text": undefined,
    }
  }
  accessible={true}
  collapsable={false}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    {
      "borderBottomWidth": 1,
      "borderColor": "#bbbbbb",
      "flexDirection": "row",
      "justifyContent": "space-between",
      "opacity": 1,
      "paddingHorizontal": 8,
      "paddingVertical": 12,
    }
  }
>
  <View
    style={
      {
        "flexDirection": "column",
        "marginRight": 4,
      }
    }
  >
    <Text
      style={
        {
          "color": "#000000",
          "fontSize": 18,
          "fontWeight": "bold",
          "marginBottom": 8,
        }
      }
    >
      Berlin
    </Text>
    <Text
      style={
        {
          "alignSelf": "flex-start",
          "color": "#555555",
        }
      }
    >
      Berlin
    </Text>
  </View>
  <Text
    style={
      {
        "color": "#6db0f7",
      }
    }
  >
    Germany
  </Text>
</View>
`;
